### PR TITLE
[NEW] module to generate sequences to the categories

### PR DESCRIPTION
**Pull request by mileo**

### CODE REVIEW

File: product_sequence_categ/model/product_product.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+from openerp.osv import orm, fields
+
+
+class ProductProduct(orm.Model):
+    _inherit = 'product.product'
+
+    def create(self, cr, uid, vals, context=None):
+
+        obj_categ = self.pool.get('product.category')
```
Comment (pedrobaeza) [inline product_sequence_categ/model/product_product.py:28]: Why overwrite product create? The sequence is for categories

File: product_sequence_categ/model/product_product.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+
+class ProductCategory(orm.Model):
+    _inherit = "product.category"
+
+    def create(self, cr, uid, vals, context=None):
+        ref = ""
+        poll_sequence = self.pool.get('ir.sequence')
+
+        if vals['parent_id']:
```
Comment (pedrobaeza) [inline product_sequence_categ/model/product_product.py:62]: I don't get why do you want to put a sequence for each parent category.

**Comment by pedrobaeza** [comment]
Thanks for the contribution, but I don't see clear the purpose of the module with the weird sequence and assignation code.

**Comment by mileo** [comment]
Example:

All products
All products / Seallable : ref S , sequence 000, next 1  
All products / Seallable / Computers : ref  S001, sequence C000, next 1 
All products / Seallable / Printers : ref  S002, sequence P000, next 1 

Product:
DELL XPS  
ref S001C001

Printer EPSON TP
ref S002P001

File: product_sequence_categ/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+# You should have received a copy of the GNU General Public License           #
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.       #
+###############################################################################
+
+{
+    'name': 'Product Sequence Category',
+    'version': '0.1',
+    'category': 'Generic Modules',
+    'description': """Generate product sequence reference based on sequence from category""",
+    'author': 'KMEE',
```
Comment (gurneyalex) [inline product_sequence_categ/__openerp__.py]: please add "Odoo Community Association (OCA)" to the list of authors.

File: product_sequence_categ/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+#                                                                             #
+# You should have received a copy of the GNU General Public License           #
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.       #
+###############################################################################
+
+{
+    'name': 'Product Sequence Category',
+    'version': '0.1',
+    'category': 'Generic Modules',
+    'description': """Generate product sequence reference based on sequence from category""",
```
Comment (gurneyalex) [inline product_sequence_categ/__openerp__.py:24]: Please move the description of the module to a `README.rst` file using the OCA template.

Comment (mileo) [inline product_sequence_categ/__openerp__.py:24]: It's a description like this? 
https://github.com/OCA/product-attribute/blob/8.0/product_brand/README.rst

Is there any guide about how to contribute? Is this http://odoo-community.org/page/how-to?

**Comment by pedrobaeza** [comment]
This has Travis red and no news from a lot, so I close it. Please feel to free to open a new one if there's still interest.

**Pull request closed without merging**